DAQ_FW/lib/Logger/Logger.cpp

```diff
@@ -0,0 +1,81 @@
+#include "Logger.h"
+
+using namespace Logger;
+
+// Output Log Template:
+// TIME [LOG_LEVEL] : LOG INFO STRING
+// 17:20:10:38 [Info] : ArduinoLog System Start Success
+
+// uses printf-like structure, ie: Error(format string, data_variables)
+
+// Wrapper function to be called at setup
+// Sets up the log message template
+void Logger::Start()
```

Comment:
```suggestion
void Start()
```

Reply:
This change causes an error in the main file, as `Logger::Start()` is now undefined.
It will work if I put `Start()` outside of the namespace, but now the call in main will not be explicit.

Is there a better way to go about this?

Reply:
Huh just tried it myself; you're right. Let's leave it as `Logger::Start` and remove the `using namespace Logger;` since it's now redundant.

For reference, I've used it in the context of nested namespaces where it does work and made sense for me. 

header file 

```
namespace one{
    namespace two{
        void Start();
    }
}
```

source 
```
#include header

using namespace one;

void two::Start()
{
    //....
}


```

I think the compiler assumes you are referring to the global namespace unless there is an explicit scope operator (`::`). In your case, the compiler actually creates a function prototype in the Logger namespace and a function definition in the global namespace. 

In my example, even though we use `using namespace one;`, we still have an explicit namespace call that tells the compiler that we are not referring to the global namespace.

Reply:
Sounds good, I pushed the change.